docs/pages/link.js

```diff
@@ -222,11 +106,40 @@ function TabExample() {
     </Box>
   );
 }`}
-      />
-      <Example
-        id="Permutations"
-        name="Permutations: tapStyle and hoverStyle"
-        defaultCode={`
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Link and Text"
+          description="You should wrap Link inside of a [Text](https://gestalt.pinterest.systems/text) component to get the correct font & underline color."
```

Comment:
This is a little awkward. Perhaps
"Use Link within Text to get the correct font and underline color"

Reply:
DONE